{"uuid": "d5cb1323-ea5a-4f31-bb78-3c3986742b95", "children": ["7134dc1d-ccf2-4701-9c1b-fe10fb35f0f8"], "befores": [{"name": "_xunit_setup_method_fixture_TestLogin", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"E:\\Python\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"E:\\Python\\lib\\site-packages\\_pytest\\fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"E:\\Python\\lib\\site-packages\\_pytest\\fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"E:\\Python\\lib\\site-packages\\_pytest\\python.py\", line 893, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"E:\\Python\\lib\\site-packages\\_pytest\\python.py\", line 778, in _call_with_optional_argument\n    func()\n  File \"E:\\Python_Learns\\WEB自动化测试框架/..\\WEB自动化测试框架\\test_case\\test_login.py\", line 23, in setup\n    self.main = Main()\n  File \"E:\\Python_Learns\\WEB自动化测试框架/..\\WEB自动化测试框架\\page\\basepage.py\", line 47, in __init__\n    self._driver.get(self.base_url)\n  File \"E:\\Python\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 333, in get\n    self.execute(Command.GET, {'url': url})\n  File \"E:\\Python\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 319, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"E:\\Python\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 374, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"E:\\Python\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 397, in _request\n    resp = self._conn.request(method, url, body=body, headers=headers)\n  File \"E:\\Python\\lib\\site-packages\\urllib3\\request.py\", line 79, in request\n    method, url, fields=fields, headers=headers, **urlopen_kw\n  File \"E:\\Python\\lib\\site-packages\\urllib3\\request.py\", line 170, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n  File \"E:\\Python\\lib\\site-packages\\urllib3\\poolmanager.py\", line 375, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"E:\\Python\\lib\\site-packages\\urllib3\\connectionpool.py\", line 706, in urlopen\n    chunked=chunked,\n  File \"E:\\Python\\lib\\site-packages\\urllib3\\connectionpool.py\", line 445, in _make_request\n    six.raise_from(e, None)\n  File \"<string>\", line 3, in raise_from\n  File \"E:\\Python\\lib\\site-packages\\urllib3\\connectionpool.py\", line 440, in _make_request\n    httplib_response = conn.getresponse()\n  File \"E:\\Python\\lib\\http\\client.py\", line 1344, in getresponse\n    response.begin()\n  File \"E:\\Python\\lib\\http\\client.py\", line 306, in begin\n    version, status, reason = self._read_status()\n  File \"E:\\Python\\lib\\http\\client.py\", line 267, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"E:\\Python\\lib\\socket.py\", line 589, in readinto\n    return self._sock.recv_into(b)\n"}, "start": 1663422100885, "stop": 1663422111942}], "start": 1663422100885, "stop": 1663422112608}